cmake/modules/FindOpenCL.cmake

```diff
@@ -0,0 +1,184 @@
+# Distributed under the OSI-approved BSD 3-Clause License.  See accompanying

```

Comment:
Where did you get this file from? Isn't this the same file as the one that comes with CMake? Why do we need it here?

Reply:
The one in CMake doesn't works properly under win32， I have to copy and modify